### PR TITLE
 Improve shortcodes definition structure in wpml-config.xml

### PR DESCRIPTION
**Pull request by strategio**
The improve structure is as below:

```xml
<!-- Existing shortcodes definition (for backward compatibility)-->
<shortcode>
    <!-- Translate string in the content -->
    <!-- Only in strings settings -->
    <tag>et_pb_foo</tag>
</shortcode>
<shortcode>
    <!-- Translate link attribute as a string, translate string in the content -->
    <!-- Only in strings settings -->
    <tag>et_pb_bar</tag>
    <attributes>
        <attribute type="link">link</media-attribute>
    </attributes>
</shortcode>

<!-- New shortcodes definition -->
<shortcode>
    <!-- Convert media IDs in `gallery_ids` attribute, and ignore the shortcode content -->
    <!-- Only in media settings -->
    <tag ignore-content="1">et_pb_gallery</tag>
    <attributes>
        <attribute type="media-ids">gallery_ids</media-attribute>
    <attributes>
</shortcode>
<shortcode>
    <!-- Convert the media URL in the content, translate some string attributes, convert media IDs in `src` attribute -->
    <!-- Both in strings and media settings -->
    <tag type="media-url">et_pb_image</tag>
    <attributes>
        <attribute>title_text</attribute>
        <attribute>alt</attribute>
        <attribute type="link">link</attribute>
        <attribute type="media-ids">src</attribute>
    </attributes>
</shortcode>
```

I added as second commit to simplify the code in the strings settings. Indeed we were having 2 separate logics for single and multiple attributes which was making the code hard to follow and debug.

This goes with some changes in the XSD file: https://git.onthegosystems.com/wpml/sitepress-multilingual-cms/merge_requests/2081.

https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlcore-5867

### CODE REVIEW

File: src/media/shortcodes/class-wpml-page-builders-media-shortcodes.php [hunk 1]
```diff
@@ -3,8 +3,8 @@
 class WPML_Page_Builders_Media_Shortcodes {
 
 	const ALL_TAGS = '\w+';
```
Comment (brucepearson) [inline src/media/shortcodes/class-wpml-page-builders-media-shortcodes.php]: @strategio I don't think is used anymore.

Comment (strategio) [reply src/media/shortcodes/class-wpml-page-builders-media-shortcodes.php]: Done @brucepearson (see https://github.com/OnTheGoSystems/wpml-page-builders/pull/33/commits/464fa49bf120f96cbbcc4bd346ff02a9bd9ac0d7). I removed the `*` wildcard  and the `|` which were giving the ability to match several tag names.
